### PR TITLE
bri anchor platform review

### PR DESCRIPTION
**Pull request by briwylde08**

### CODE REVIEW

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx [hunk 1]
```diff
@@ -590,30 +592,32 @@ After KYC is collected, you should be ready to receive the off-chain deposit. Wh
...
+Please note that you **must** specify amount fields, as wallets do not provide this information.
 
-- `amount_in` is amount user requested. If `amount_expected` is not null, `amount_in` should be equal to `amount_expected`. In that case, amount should not be collected in the interactive flow.
-- `amount_out` is amount user will receive
-- `amount_fee` is total fees collected by the application.
-- `asset` part of amount fields is an [SEP-38][sep-38] formatted asset of the amount. In this example, it's set to USD, assuming user made a bank transfer to the system using USD.
+- `amount_in` is the amount the user requested. If `amount_expected` is not null, `amount_in` should be equal to `amount_expected`. In that case, the amount should not be collected in the interactive flow.
+- `amount_out` is the amount the user will receive.
+- `amount_fee` is the total fees collected by the application.
+- `asset` is part of the amount fields is a [SEP-38][sep-38] formatted asset of the amount. In this example, it's set to USD, assuming the user made a bank transfer to the system using USD.
```
Comment (briwylde08) [inline docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:602]: The wording for asset is a little confusing here. Do you mean: "'asset' is part of the amount field in a SEP-38 formatted asset..." Or how can we change this wording to make sense?

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx [hunk 2]
```diff
@@ -590,30 +592,32 @@ After KYC is collected, you should be ready to receive the off-chain deposit. Wh
...
 
-This step is optional. Most business don't use it: you can skip it and go to the [next one](#sending-stellar-transaction)
+This step is optional. Most businesses don't use it: you can skip it and go to the [next step](#sending-stellar-transaction)
 
 You should only use this status if the off-chain payment method being used notifies you of a pending inbound payment.
 
 :::
 
-After update in the previous chapter has been made, user will be notified that your service is ready to receive funds. In real world, transfer confirmation may take time. In such cases, transaction should be set to a new status, after funds have been received, but not confirmed yet.
+After the update in the previous chapter has been made, the user will be notified that your service is ready to receive funds. In the real world, transfer confirmation may take time. In such cases, transaction should be set to a new status, after funds have been received, but not confirmed yet.
```
Comment (briwylde08) [inline docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:620]: Wording here is also a little confusing. Does this mean that the funds have been received but not confirmed? If so, maybe we can say: "In the real world, transfer confirmation may take time. If the funds have been received but not confirmed, the transaction should be set to a new status."

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx [hunk 3]
```diff
@@ -766,37 +770,37 @@ Similar to deposit, the next step would be to notify the user that the Anchor is
...
 
 :::
 
 :::info
 
-If you already have system that monitors payments, make sure that the logic of the system matches the description below:
+If you already have a system that monitors payments, make sure that the logic of the system matches the description below:
 
-First, wait for the transaction to be included in the ledger (using SDK). This transaction must have correct memo and destination address (distribution account). Once this transaction has been detected and verified, notify the user that the funds have been received:
+First, wait for the transaction to be included in the ledger (using an  SDK). This transaction must have the correct memo and destination address (distribution account). Once this transaction has been detected and verified, notify the user that the funds have been received:
```
Comment (briwylde08) [inline docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:803]: What SDK? Any SDK?

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx [hunk 4]
```diff
@@ -1259,11 +1263,11 @@ async function getPlatformTransaction(transactionId) {
 
 ### Full Example Implementation
 
-Stellar provides an example business server implementation for SEP-24. It's split into 2 parts: web UI, accessible for the end user and back-end implementation, used to get and push updates from/to the Anchor Platform.
+Stellar provides an example business server implementation for SEP-24. It's split into two parts: a web UI, accessible for the end user and back-end implementation, used to get and push updates from/to the Anchor Platform.
```
Comment (briwylde08) [inline docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:1266]: It's not clear what the two parts are here. I would recommend doing something like: "It's split into two parts: 1) [insert first part here]; and 2) [insert second point here].

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx (hunk 2 above)
Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:620]: Yeah, this part is worded incorrectly. It means:
Transfer is in process. Confirmation of the transfer has been received, but funds itself may not been received yet

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx [hunk 5]
```diff
@@ -766,37 +770,37 @@ Similar to deposit, the next step would be to notify the user that the Anchor is
...
 :::
 
-##### Tracking Stellar Transaction
+##### Tracking Stellar Transactions
 
-Using Payment Observer allows delegating this step to the Anchor Platform. To enable Payment Observer use `--stellar-observer` flag in the command section of the [compose file](#configuration)
+Using the Payment Observer allows you to delegate this step to the Anchor Platform. To enable the Payment Observer use the `--stellar-observer` flag in the command section of the [compose file](#configuration).
 
-Payment Observer will track all transactions sent to distribution account. When transaction with correct memo is detected in the network, status will automatically change to `pending_anchor` and event will be emitted (if Kafka is used).
+The Payment Observer will track all transactions sent to the distribution account. When the transaction with the correct memo is detected in the network, the status will automatically change to `pending_anchor` and event will be the omitted (if Kafka is used).
```
[the remark this answers is not in the document]
Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:791]: It's a common phrase to ["emit the event"](https://stackoverflow.com/questions/13438924/what-is-an-event-emitter)

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx (hunk 4 above)
Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:1266]: 1) web UI, accessible for the end user
2) back-end implementation, used to get and push updates from/to the Anchor Platform
Something like that?

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx (hunk 1 above)
Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:602]: Yeah, unfortunately I couldn't make it more clear. 
Asset if a part of `amount_x` fields (amount_in/out/fee). And it's in a SEP-38 **format**. 
(That's what I was trying to convey at least 😅 )

Comment (briwylde08) [reply docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:602]: does this make sense then?:  `asset` is part of the `amount_x` field and is in a SEP-38 format.

Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:602]: Yes!

**Review by Ifropc — APPROVED** [review]

**Pull request merged**